rpc/server.go

```diff
@@ -122,3 +158,39 @@ func (s *Server) RegisterService(name string, service interface{}) error {
 func (s *Server) Stop() {
 	close(s.stopCh)
 }
+
+func handleCipher(conn net.Conn) (cryptoConn *etls.CryptoConn, err error) {
+	// NodeID + Uint256 Nonce
+	headerBuf := make([]byte, hash.HashBSize+32)
+	rCount, err := conn.Read(headerBuf)
+	if err != nil || rCount != hash.HashBSize+32 {
+		log.Errorf("read node header error: %s", err)
+		return
+	}
+
+	// nodeIdBuf len is hash.HashBSize, so there won't be any error
+	idHash, _ := hash.NewHash(headerBuf[:hash.HashBSize])
+	nodeId := proto.NodeID(idHash.String())
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var nodeId should be nodeID